app/assets/javascripts/discourse/app/controllers/application.js

```diff
@@ -94,8 +94,8 @@ export default class ApplicationController extends Controller {
       return false;
     }
 
-    // Always show sidebar for admin if user can see the admin sidbar
-    if (this.sidebarState.isForcingAdminSidebar) {
+    // Always show sidebar for admin if user can see the admin sidebar
```

Comment:
martin is working on his spelling :)

Reply:
![dumb-and-dumber-jim-carrey](https://github.com/user-attachments/assets/99769770-ca3b-42a0-ab70-c61ae8f0ff90)